README.md

```diff
@@ -4,9 +4,9 @@
 Facilitator: @rsimha(https://github.com/rsimha)
 
 # Members
-- [@ajwhatson](https://github.com/ajwhatson) - Anna Watson
```

Comment:
This makes me wonder if we should add a "Past contributors" section to give attribution to Anna, Erwin, Marcial, etc.

Reply:
Great idea. UI does this: https://github.com/ampproject/wg-ui-and-a11y#past-members

Reply:
+1 great idea. Added.